transport.go

```diff
@@ -98,9 +103,39 @@ func addWebhookRequestDecoder(config transportConfig) kithttp.DecodeRequestFunc
 
 		wv.setWebhookDefaults(&webhook, r.RemoteAddr)
 
+		auth, _ := bascule.FromContext(c)
+		var partners []string
+
+		switch auth.Token.Type() {
+		case "Basic":
+			var configurableString string
```

Comment:
We can add this string (I would name it something like `basicPartnerIDsHeader` to the `transportConfig` that we pass into the create decoder function.  Then, if it's empty, we should set it to a const called `DefaultBasicPartnerIDsHeader`.  I would add this empty string check above the creation of this function to return (ie around line 82).